### ISSUE TITLE
VERSION_NAME and VERSION_CODE not set in iOS

### ISSUE THREAD

**Issue by jehartzog** [react-native-config/react-native-config#256]
Though not super clear in the docs, this package [pre-populates some useful variables](https://github.com/luggit/react-native-config/issues/205#issuecomment-375152771).

These work as expected on Android, but on iOS both `VERSION_NAME` and `VERSION_CODE` were undefined.

I believe these should come from the `Bundle versions string, short` and `Bundle version`, which are set in the project I am using.

**Comment by iadcg** [comment]
Hi @jehartzog . Same problem here. Were you able to get this values for iOS?

**Comment by jehartzog** [comment]
@iadcg No, I ended up using a specific `IOS_VERSION_STRING` and `IOS_BUILD_NUMBER` values and used a wrapper to grab the correct one for iOS vs android.

**Comment by iadcg** [comment]
@jehartzog Sorry mate. I lost you there. Didnt get what you mean by using a wrapper. Im new to react native + iOS kinda thing. Can explain a bit more?
